forte/data/data_store.py

```diff
@@ -422,8 +428,13 @@ def _is_annotation(self, type_name: str) -> bool:
             type or not.
         """
         # TODO: use is_subclass() in DataStore to replace this
-        entry_class = get_class(type_name)
-        return issubclass(entry_class, (Annotation, AudioAnnotation))
+        try:
+            return self._is_annotation_cache[type_name]
```

Comment:
so according to our discussion, we don't need this cache, we can rely on the internal issubclass